### ISSUE TITLE
Conflicted copies happen even when only using phone

### ISSUE THREAD

**Issue by timgalebach** [standardnotes/mobile#157]
When using my phone client on a connection with intermittent access, I frequently end up with conflicted copies. I assume this shouldn't happen if I'm only on one device?

Steps to reproduce: edit a note several times over the course of an hour or so, generally in a location with spotty internet access (gym, concrete building, etc).

**Comment by mobitar** [comment]
What version of the app is this with, and what OS?

**Comment by timgalebach** [comment]
3.0.10, on iOS

**Comment by mobitar** [comment]
I tested typing for about 5 minutes on a network that simulates 3G speeds. Was unable to produce conflicts. Would you say your network was really slow, or it would drop requests all together at some point and show an error saving?

**Comment by timgalebach** [comment]
It would drop requests altogether at times. It seems like that shouldn’t be an issue if all editing is on one client though, no?

I LOVE Standard Notes and am a subscriber, so this isn’t a deal breaker by any means. Just an annoyance that comes up a decent amount.

**Comment by mobitar** [comment]
Well, conflicts are created based on if the item you're trying to save has the same timestamp as what the server has. If requests are being dropped, then it may be that the server begins to have different timestamps than the local value, thus causing conflicts in future saves. This would seem to happen if the server successfully performs a save but the client drops the response that would keep its item in sync.

**Comment by timgalebach** [comment]
Yeah, that’s likely the cause. Wouldn’t the more intuitive behavior be to check whether the client copy is a fast-forward of the parent copy, in a git sense?

**Comment by mobitar** [comment]
We don't really track revisions on the client or server. It's just 1 final copy of the item we hang on to.
